src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -216,6 +216,42 @@ public async Task ProcessActionWithBotIdentifierHeaderShouldSucceed()
                     h => h.RequestUri.Equals(settings.Uri)), Arg.Any<CancellationToken>());
         }
 
+        [Fact]
+        public async Task ProcessActionTimeoutShouldSucceed()
+        {
+            // Arrange
+            var settings = new ProcessHttpSettings
+            {
+                Uri = new Uri("https://blip.ai"),
+                Method = HttpMethod.Post.ToString(),
+                Body = "{\"plan\":\"Premium\",\"details\":{\"address\": \"Rua X\"}}",
+                Headers = new Dictionary<string, string>()
+                {
+                    {"Content-Type", "application/json"},
+                    {"Authorization", "Key askçjdhaklsdghasklgdasd="}
+                },
+                ResponseBodyVariable = "httpResultBody",
+                ResponseStatusVariable = "httpResultStatus",
+
+            };
+
+            var target = GetTarget();
+
+            HttpClient.SendAsync(Arg.Any<HttpRequestMessage>(), Arg.Any<CancellationToken>())
+                .Returns(async x =>
```

Comment:
Replace unused variable with discard

```suggestion
                .Returns(async _ =>
```

Reply:
Changed variable name to token.